src/app/components/MobileFooterNavigation/index.tsx

```diff
@@ -23,15 +25,24 @@ const StyledMobileFooterNavigation = styled.nav`
   right: 0;
   height: ${mobileFooterNavigationHeight};
   display: flex;
-  align-items: center;
-  justify-content: space-evenly;
+  align-items: stretch;
+  justify-content: space-between;
   box-shadow: ${({ theme }) =>
     `0px 0px ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-front-border', theme)}`};
   border-top: ${({ theme }) =>
     `solid ${theme.global?.borderSize?.xsmall} ${normalizeColor('background-contrast', theme)}`};
   flex-direction: row;
 `
 
+const StyledNavLink = styled(NavLink)`
+  // Make items mostly-equal width
```

Comment:
This comment has no context now

Reply:
should be "Make items equal width" now

Reply:
I think comment is not needed at all, as it is straight forward from the code.

Reply:
`flex-basis: 0;` seemed unintuitive to me :/ I wouldn't need a comment if it was `grid-template-columns: 1fr 1fr 1fr 1fr;`